{"uuid":"916d613e-9e91-4a27-ac57-47aec942ef54","historyId":"2526eac396044ac2746e0ae01a5c2bd8","fullName":"allureReporting.testcases.TestCases.doUserReg","labels":[{"name":"package","value":"allureReporting.testcases.TestCases"},{"name":"testClass","value":"allureReporting.testcases.TestCases"},{"name":"testMethod","value":"doUserReg"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"allureReporting.testcases.TestCases"},{"name":"host","value":"Kunals-MacBook-Pro.local"},{"name":"thread","value":"5516@Kunals-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Registration Functionality"},{"name":"feature","value":"TestNG support"},{"name":"epic","value":"Allure example"},{"name":"severity","value":"blocker"}],"links":[],"name":"doUserReg","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Failing test case","trace":"java.lang.AssertionError: Failing test case\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat allureReporting.testcases.TestCases.doUserReg(TestCases.java:38)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\n"},"stage":"finished","description":"User Registration Test","steps":[],"attachments":[],"parameters":[],"start":1753000171038,"stop":1753000171041}